analyzer-java/src/main/java/com/infosupport/ldoc/analyzerj/AnalysisVisitor.java

```diff
@@ -432,4 +433,13 @@ public List<Description> visit(JavadocComment n, Analyzer arg) {
             !commentParams.isEmpty() ? commentParams : null,
             !commentTypeParams.isEmpty() ? commentTypeParams : null));
   }
+
+  /** Describes a Variable Declaration Expression. As these are not supported in the JSON, there is
```

Comment:
Nit: Prefer starting 'Describes' on the next line, for consistency with multi-line Javadoc comments elsewhere.